### DISCUSSION TITLE
Is it possible to have a minimal customization of the User Form?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11066] [Archive]
Is there a way to hide certain fields in the User Detail? 
`Token` is an example. Showing it will bring many not tech users to ask "_What is a token?_" etc.  and this will generate a big loss of time explaining them that it is of no use to them.
`Language` is another example when the users are all of the same language.
etc.
Apart the compulsory fields, it would be nice to hide some of the Directus basic fields that are not useful.

Also, is there a way to move them or make them Half-Width?
Or to place some custom added fields to the Directus basic fields within the  Directus basic fields zone?

**Comment by benhaynes** [comment]
No way to change those natively as of now, but open to discussing how Core Fields could be hidden.

**Comment by joselcvarela** [comment]
Hello @Pachat 
While you cannot hide fields for admin users you can hide for users that are in a role (I think @benhaynes forgot about Field Permissions 😅)

In your case you would do something like this:

https://user-images.githubusercontent.com/14039341/149627616-1e0e3f0d-26c2-49c5-afdd-73c225b377fa.mov

It seems only certain fields can be hidden like `token`, while `language` is not. Regarding this, you may want to create an issue instead 👍

**Reply by benhaynes** [reply]
Very true! Thanks, @joselcvarela!

**Reply by Pachat** [reply]
This satisfies my needs. Looks like `Language `is among the required fields that are necessary for the Directus App
